documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus
+  name: datagrid-monitoring # Names the service monitor.
+  namespace: infinispan-monitoring # Namespace of the Prometheus stack.
+spec:
+  endpoints:
+    - targetPort: 11223 # Port where {brandname} exposes the metrics endpoint.
+      path: /metrics # Path for the {brandname} metrics endpoint.
+      honorLabels: true
+      basicAuth:
+        username:
+          key: username # Username field in the authentication secret.
+          name: basic-auth # Name of the authentication secret.
+        password:
+          key: password # Password field in the authentication secret.
+          name: basic-auth # Name of the authentication secret.
+      interval: 30s
+      scrapeTimeout: 10s
+      scheme: http # Do not use encryption.
+      tlsConfig:
+        insecureSkipVerify: true
+        serverName: {example_crd_name} # Name of your {brandname} cluster.
```

Comment:
No need to configure `tlsConfig` with the introduction of the 11223 operator port as https is not enabled.